storage/remote/queue_manager.go

```diff
@@ -352,10 +352,12 @@ type QueueManager struct {
 	clientMtx   sync.RWMutex
 	storeClient WriteClient
 
-	seriesMtx            sync.Mutex
-	seriesLabels         map[uint64]labels.Labels
+	seriesMtx     sync.Mutex // covers seriesLabels and droppedSeries
```

Comment:
nit; this comment and the one for `seriesSegmentMtx` should start with capitals and end in `.`